Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -281,6 +281,25 @@ private void OnFocusChanged(bool focus)
         public Vector2 screenSize => new Vector2(Screen.width, Screen.height);
         public ScreenOrientation screenOrientation => Screen.orientation;
 
+        public bool normalizeScrollWheelDelta
+        {
+#if UNITY_6000_0_OR_NEWER
+            get => NativeInputSystem.normalizeScrollWheelDelta;
+            set => NativeInputSystem.normalizeScrollWheelDelta = value;
+#else
+            get; set;
```

Comment:
Not sure I see what role these getter setters play when earlier Unity version? Wouldn't it make sense to remove the property completely instead?

Reply:
(or would have expected hard coded behavior, but the answer is - it depends)